docs/developers/README.md

```diff
@@ -88,10 +88,10 @@ expected in directory path `/workspace/output/resource_name`.
   inputs:
     resources:
       - name: gcs-workspace
-    type: storage
-    name: storage
+        type: storage
+        name: storage
```

Comment:
`name` key is repeated here. I notice that its already in old yaml file but lets take this opportunity to fix this